### PR TITLE
chore: Update `PersonPicture` to winui3/release/1.4.2

### PR DESCRIPTION
**Pull request by morning4coffe-dev**
GitHub Issue (If applicable): part of #14275

## PR Type

What kind of change does this PR introduce?

- Code style update (formatting)
- Refactoring (no functional changes, no api changes)

## What is the new behavior?

Updated the `PersonPicture` code to match the new public winui3/release/1.4.2

### <samp>🤖 Generated by Copilot at a63fbc5</samp>

This pull request updates the `PersonPicture` control and its related classes to match the latest version of the WinUI 3.0 library. It also adds accessibility support and merges the theme resources into one file.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml [hunk 1]
```diff
@@ -1,168 +1,84 @@
...
+                    <Setter Target="BadgingEllipse.Fill">
+                      <Setter.Value>
+                        <ImageBrush x:Name="BadgeImageBrush" Stretch="UniformToFill" />
+                      </Setter.Value>
+                    </Setter>
+                  </VisualState.Setters>
+                </VisualState>
+              </VisualStateGroup>
+            </VisualStateManager.VisualStateGroups>
+            <Ellipse Fill="{ThemeResource PersonPictureEllipseFillThemeBrush}" Stroke="{ThemeResource PersonPictureEllipseFillStrokeBrush}" StrokeThickness="{ThemeResource PersonPictureEllipseStrokeThickness}" Width="{TemplateBinding Width}" Height="{TemplateBinding Height}" />
```
Comment (Youssef1313) [inline src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml:65]: I think we should be reverting this and keep the `TemplateBinding` code? @MartinZikmund thoughts?

File: src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml [hunk 2]
```diff
@@ -1,168 +1,84 @@
...
+        <ControlTemplate TargetType="controls:PersonPicture">
+          <Grid x:Name="RootGrid">
+            <VisualStateManager.VisualStateGroups>
+              <VisualStateGroup x:Name="CommonStates">
+                <!-- Visual State when a Photo is available for display -->
+                <VisualState x:Name="Photo">
+                  <VisualState.Setters>
+                    <Setter Target="PersonPictureEllipse.Fill" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ActualImageBrush}" />
+                    <!-- UNO WORKAROUND: ImageBrush is not stretching properly https://github.com/unoplatform/uno/issues/4951 -->
+                    <wasm:Setter Target="PersonPictureImage.Source" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ActualImageBrush.ImageSource}" />
```
Comment (Youssef1313) [inline src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml:24]: This wasm-specific workaround might no longer be needed. Consider removing it and see if everything is working as expected.

Comment (morning4coffe-dev) [reply src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml:24]: @Youssef1313 Indeed seems to be fixed, I haven't noticed that the second ongoing referenced PR is not for wasm. Thanks!

File: src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml (hunk 1 above)
Comment (MartinZikmund) [reply src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml:63]: You mean reverting the workaround?

Comment (MartinZikmund) [reply src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml:63]: Ah yes, we should include it, Microsoft omitted it from WinUI3 probably by accident

**Review by MartinZikmund — APPROVED** [review]

**Pull request merged**